### PR TITLE
Added chatmate for facebook

### PR DESCRIPTION
**Pull request by SebTM**
*If there’s a checkbox you can’t complete for any reason, that's okay, just explain in detail why you weren’t able to do so.*

After making all changes to the cask:

- [X] `brew cask audit --download {{cask_file}}` is error-free.
- [ ] `brew cask style --fix {{cask_file}}` reports no offenses.
Can fix this, the download-url differs from the website-url.

- [ ] The commit message includes the cask’s name and version.
Hmm, I think I missed this - it's "chatmate-for-facebook" with version "latest".

Additionally, **if adding a new cask**:

- [X] Named the cask according to the [token reference].
- [X] `brew cask install {{cask_file}}` worked successfully.
- [X] `brew cask uninstall {{cask_file}}` worked successfully.
- [X] Checked there are no [open pull requests] for the same cask.
- [X] Checked the cask was not already refused in [closed issues].
- [X] Checked the cask is submitted to [the correct repo].
I think it should be right, app is install- and testable for free (demoversion) but can be activated with license-key after puchase.

[token reference]: https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md
[open pull requests]: https://github.com/caskroom/homebrew-cask/pulls
[closed issues]: https://github.com/caskroom/homebrew-cask/issues?q=is%3Aissue+is%3Aclosed
[the correct repo]: https://github.com/caskroom/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask

### CODE REVIEW

**Comment by SebTM** [comment]
@vitorgalvao I failed with adding "chatmate-for-whats" to another changeset to make a seperate PR. Can you split "chatmate-for-whats" or merge together? thx

File: Casks/chatmate-for-whats.rb [hunk 1]
```diff
@@ -0,0 +1,19 @@
+cask 'chatmate-for-whats' do
```
Comment (vitorgalvao) [inline Casks/chatmate-for-whats.rb]: Name it `chatmate-for-whatsapp`

File: Casks/chatmate-for-whats.rb [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+  version :latest
+  sha256 :no_check
+
+  url 'https://dl.devmate.com/net.coldx.mac.WhatsApp/ChatMateforWhatsApp.dmg'
```
Comment (vitorgalvao) [inline Casks/chatmate-for-whats.rb]: This needs a comment, like the other one.

**Review by vitorgalvao — CHANGES_REQUESTED** [review]

**Comment by vitorgalvao** [comment]
In the future, please submit separate pull requests for each cask, as more often than not this type of pull request brings problems with one or more of them, and it halts the inclusion of the other ones.

Also, do not turn off the box to allow edits from maintainers. Turning that off just makes our lives harder and it’ll take longer to merge your PR.

**Pull request closed without merging**